extensions/amp-accordion/1.0/accordion.js

```diff
@@ -82,6 +88,18 @@ export function Accordion({
     });
   }, [expandSingleSection]);
 
+  useImperativeHandle(
+    ref,
+    () => {
+      return {
+        registerSection,
```

Comment:
Just one note that, `toggleExpanded` and `isExpanded` could be right here. But `registerSection` is definitely out of place. Think about it this way: the imperative API (along with component props) is part of the component's public API. The `registerSection` is an internal detail.

Reply:
Definitely makes sense, thanks Dima!